lib/components/home/artist_rails/artist_card.js

```diff
@@ -8,19 +8,26 @@ const { ARTemporaryAPIModule } = NativeModules
 
 import colors from '../../../../data/colors'
 import ImageView from '../../opaque_image_view'
-import SwitchBoard from '../../../modules/switch_board'
+import SwitchBoard from '../../../native_modules/switch_board'
 import InvertedButton from '../../buttons/inverted_button'
-import Events from '../../../modules/events'
+import Events from '../../../native_modules/events'
+
+// $FlowFixMe: Flow doesn't know RN provides this
+import { Promise } from 'react-native'
```

Comment:
long discussion about this, still ongoing. However, it is needed here to give a bool later on.